crates/iota-core/src/authority/consensus_quarantine.rs

```diff
@@ -817,6 +900,85 @@ impl ConsensusOutputQuarantine {
             .filter_map(|output| output.get_randomness_last_round_timestamp())
             .next()
     }
+
+    pub(crate) fn load_initial_object_debts(
+        &self,
+        epoch_store: &AuthorityPerEpochStore,
+        current_round: Round,
+        for_randomness: bool,
+        transactions: &[VerifiedSequencedConsensusTransaction],
+    ) -> IotaResult<impl IntoIterator<Item = (ObjectID, u64)>> {
+        let protocol_config = epoch_store.protocol_config();
+        let tables = epoch_store.tables()?;
+        let default_per_commit_budget = protocol_config
+            .max_accumulated_txn_cost_per_object_in_mysticeti_commit_as_option()
+            .unwrap_or(0);
```

Comment:
```suggestion
            .unwrap_or_default();
```

This is more idiomatic.